recipes/rmlui/4.x/conanfile.py

```diff
@@ -31,7 +36,6 @@ class RmluiConan(ConanFile):
     }
     build_requires = ["cmake/3.23.2"]
     exports_sources = ["CMakeLists.txt"]
```

Comment:
I think you should be able to remove this after the migration you've done. You can also delete the file it would've copied.